scripts/readelf.py

```diff
@@ -1552,10 +1552,24 @@ def _dump_debug_locsection(self, di, loc_lists_sec):
             # Present but empty locations section - readelf outputs a message
             self._emitline("\nSection '%s' has no debugging data." % (section_name,))
             return
+        
+        lcus = list(loc_lists_sec.iter_CUs()) if ver5 else None
```

Comment:
It'd be nice to have some comments here explaining what this is doing

Reply:
Commented. Same logic was under rnglists for a while.